src/views/tutorial-view/index.tsx

```diff
@@ -266,6 +267,12 @@ function TutorialView({
 							}}
 							tutorialId={id}
 						/>
+						{metadata.variant ? (
+							<div style={{ border: '1px solid magenta' }}>
+								<h2>Variant: {metadata.variant.slug}</h2>
+								<p>Option: {metadata.variant.optionSlug}</p>
+							</div>
+						) : null}
```

Comment:
This is a stub that basically passes data from the query param. We will get real data from the API with the tutorial data.